downloader/download.go

```diff
@@ -91,9 +87,8 @@ func (d *downloader) Download(files, dirs []string, readPath, writePath string,
 	for _, val := range dirs {
 		dirWPath := writePath + filepath.FromSlash(val)
 		dirRPath := readPath + val
```

Comment:
I like this idea of thinking in terms of Read and Write paths. I think it's mentally simpler that thinking in terms of server and local paths. That's a great convention; well done.